### PR TITLE
Created ADB Add-on

### PR DESCRIPTION
**Pull request by noodlemctwoodle**
Home Assistant ADB Addon for controlling Android TV.

### CODE REVIEW

File: docs/home-automation/home-assistant/supervisor-addons/android-debug-bridge/adb.md [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+## HD Homerun
+  adb_open_shield_hd_hr:
+    sequence:
+      - service: androidtv.adb_command
+        data:
+          entity_id: media_player.shield_shield_adb
+          command: "am start -a android.intent.action.VIEW -d -n com.silicondust.view/.App"
+```
+
+Further working examples can be found on my [GitHub](https://github.com/noodlemctwoodle/homeassistant)
```
Comment (RealOrangeOne) [inline docs/home-automation/home-assistant/supervisor-addons/android-debug-bridge/adb.md:336]: Not sure I like the use of the word "my" here, feels a bit odd when there's no noted author.

```suggestion
[This repo](https://github.com/noodlemctwoodle/homeassistant) contains further working examples.
```

File: docs/home-automation/home-assistant/supervisor-addons/android-debug-bridge/adb.md [hunk 2]
```diff
@@ -0,0 +1,336 @@
+# ADB - Android Debug Bridge
+
+Requirements: 
+
+ * [ADB Addon](https://github.com/hassio-addons/addon-adb/blob/v0.6.1/README.md)
```
Comment (RealOrangeOne) [inline docs/home-automation/home-assistant/supervisor-addons/android-debug-bridge/adb.md:5]: This should probably link to the entire repo directly

```suggestion
 * [ADB Addon](https://github.com/hassio-addons/addon-adb#readme)
```

File: docs/home-automation/home-assistant/supervisor-addons/android-debug-bridge/adb.md [hunk 3]
```diff
@@ -0,0 +1,336 @@
...
+6. Allow `Home Assistant` to access to your device, remember the check the box to always remember this device
+
+
+## Connect to the container running ADB
+
+If you are using Home Assistant backed Ubuntu or other debian based OS this next step is done in Bash, if you are running the enclosed Home Assistant OS install the [Terminal & SSH plugin](https://github.com/home-assistant/hassio-addons/tree/master/ssh)
+
+Open you command line `Shell` to bash into the container
+
+    docker exec -it addon_a0d7b954_adb /bin/bash
```
Comment (RealOrangeOne) [inline docs/home-automation/home-assistant/supervisor-addons/android-debug-bridge/adb.md:52]: The container name in this example looks a bit odd, and possibly specific to your machine. Is there a canonical name for it / some way to find out its name?

File: docs/home-automation/home-assistant/supervisor-addons/android-debug-bridge/adb.md [hunk 4]
```diff
@@ -0,0 +1,336 @@
...
+    com.android.gallery3d/.app.GalleryActivity
+    com.nvidia.ota/.ui.LauncherActivity
+    com.ubnt.unifi.protect/com.ubnt.sections.splash.SplashActivity
+    com.google.android.youtube.tv/com.google.android.apps.youtube.tv.activity.ShellActivity
+    com.google.android.tv.remote.service/.settings.SettingsActivity
+    com.valvesoftware.steamlink/.SteamShellActivity
+    com.netflix.ninja/.MainActivity
+
+
+Make a list of your applications and and the options available, you'll need this later. I have exported a few Apps below for the UK users :)
```
Comment (RealOrangeOne) [inline docs/home-automation/home-assistant/supervisor-addons/android-debug-bridge/adb.md:102]: I can't see much here which is specific to UK, besides a couple of the services? 

```suggestion
Make a list of your applications and and the options available, you'll need this later.
```

File: docs/home-automation/home-assistant/supervisor-addons/android-debug-bridge/adb.md (hunk 3 above)
Comment (noodlemctwoodle) [reply docs/home-automation/home-assistant/supervisor-addons/android-debug-bridge/adb.md:52]: All the home assistant add-ons have the same name

Comment (RealOrangeOne) [reply docs/home-automation/home-assistant/supervisor-addons/android-debug-bridge/adb.md:52]: Including the hash on the end? Interesting

Comment (noodlemctwoodle) [reply docs/home-automation/home-assistant/supervisor-addons/android-debug-bridge/adb.md:52]: `addon_a0d7b954_adb` will be the same on all instances

File: docs/home-automation/home-assistant/supervisor-addons/android-debug-bridge/adb.md (hunk 2 above)
Comment (noodlemctwoodle) [reply docs/home-automation/home-assistant/supervisor-addons/android-debug-bridge/adb.md:5]: Changed URL to link to `https://github.com/hassio-addons/addon-adb/`

File: docs/home-automation/home-assistant/supervisor-addons/android-debug-bridge/adb.md (hunk 3 above)
Comment (noodlemctwoodle) [reply docs/home-automation/home-assistant/supervisor-addons/android-debug-bridge/adb.md:52]: image

two separate instances on separate hosts

File: docs/home-automation/home-assistant/supervisor-addons/android-debug-bridge/adb.md (hunk 4 above)
Comment (noodlemctwoodle) [reply docs/home-automation/home-assistant/supervisor-addons/android-debug-bridge/adb.md]: Rephrased this line

File: docs/home-automation/home-assistant/supervisor-addons/android-debug-bridge/adb.md (hunk 1 above)
Comment (noodlemctwoodle) [reply docs/home-automation/home-assistant/supervisor-addons/android-debug-bridge/adb.md]: changed sentance to `Further working examples can be found on this `

**Pull request merged**